Istio 1.23 release notes (#15538)
istio/istio.io: content/en/news/releases/1.23.x/announcing-1.23/upgrade-notes/index.md

---
title: Upgrade Notes

> description: Important changes to consider when upgrading to Istio 1.23.0

Reviewer: ```suggestion
description: Important changes to consider when upgrading to Istio 1.23.0.
```

weight: 20
publishdate: 2024-08-13
---

When you upgrade from Istio 1.22.x to Istio 1.23.x, you need to consider the changes on this page.
These notes detail the changes which purposefully break backwards compatibility with Istio 1.22.x.
The notes also mention changes which preserve backwards compatibility while introducing new behavior.
Changes are only included if the new behavior would be unexpected to a user of Istio 1.22.x.

> ## Internal API protobuf changes
> If you do not use Istio APIs from Go (via `istio.io/api` or `istio.io/client-go`) or Protobuf (from `istio.io/api`), this change does not impact you.

Reviewer: ```suggestion
## Internal API protobuf changes

If you do not use Istio APIs from Go (via `istio.io/api` or `istio.io/client-go`) or Protobuf (from `istio.io/api`), this change does not impact you.
```

publishdate: 2024-08-13
---

> When upgrading from Istio 1.22.x to Istio 1.23.x, please consider these changes

Reviewer: ```suggestion
When upgrading from Istio 1.22.x to Istio 1.23.x, please consider the changes on this page.
```
